src/test/javascript/spec/component/programming-exercise/programming-exercise-instruction.component.spec.ts

```diff
@@ -479,4 +477,89 @@ describe('ProgrammingExerciseInstructionComponent', () => {
             },
         });
     }));
+
+    it('foo (non legacy case)', fakeAsync(() => {
```

Comment:
Can you give this test case a better name?

Reply:
Oops, solved